doc/rados/operations/pools.rst

```diff
@@ -91,13 +93,14 @@ Where:
 
 .. describe:: {pg-num}
 
-   The total number of placement groups for the pool. See :ref:`placement groups`
-   for details on calculating a suitable number. The
-   default value ``8`` is NOT suitable for most systems.
+   The total number of placement groups for the pool. See :ref:`placement
+   groups` for details on calculating a suitable number. The default value
+   ``32`` is NOT suitable for most systems.
```

Comment:
Hardcoding defaults is discouraged, especially since they can change between releases.  Please replace this with something like

>The default value of :confval:`pg_num` is likely too small for production pools used for bulk data, including RBD and RGW data and bucket pools respectively.

Also, while you're here, on lines 94 and 105 please s/-/_/

Reply:
> Also, while you're here, on lines 94 and 105 please s/-/_/

That's the only thing I didn't do, it seemed inconsistent? In those places, `pg-num`/`pgp-num` refer to the arguments of the `pool create` command, and arguments seem to use hyphens throughout the entire doc?

Just guessing, though, happy to be overridden if you say so.

Reply:
Ah, I missed that.  Nevermind ;)